docs/reference/field-types/relationship.md

```diff
@@ -0,0 +1,119 @@
+:::
+
+<!-- TODO: The following settings are likely to return, but are not yet implemented. -->
+<!-- |`schema` | A schema object | n/a | In the presence of relationship, the content of a join is loaded differently. If the join is called `_departments` then `_departments[0]` will be an object with item and relationship properties, in which `item` is the department and `relationship` contains the relationship properties, rather than a flat array of departments. subproperties: `name`, `value`, `type`, `inline` | -->
+<!-- |contextual | Boolean | false | If `true`, it will prevent the field from appearing in the editor modal | -->
+<!-- |readOnly | Boolean | false | If `true`, prevents the user from editing the field value | -->
+
+## Filtering related document properties
+
+Often when two Apostrophe documents are connected by a `relationship` field, the original doc only needs one or two properties from the connected doc. For example, an `article` piece may connect to an `author` piece, but only need the author's name and portrait photo. That `author` piece may also contain rich text for a biography, an additional set of photos for a slideshow, and a string field with their home town, but we don't want to send all of that to the `article` piece as it is unnecessary and adds to work done by the server.
+
+A `projection` filter limits the properties of the connected doc that are populated on the original doc. The following configuration would limit the author data to only what we need:
+
+```javascript
+_author: {
+  label: 'Author',
+  type: 'relationship',
+  max: 1, // There's only one author
+  // Limiting our data here 👇
+  filters: {
+    projection: {
+      title: 1, // The author's name is entered as its `title`
+      photo: 1
+    }
+  }
+}
+```
+
+By doing that, we don't get any slideshow, biography, home town, or other data that isn't needed. The projection filter format comes from the similar MongoDB filter.
+
+## Limiting returned data with `ifOnlyOne`
+
+The `ifOnlyOne` option was designed to lighten document data in situations where many of that type of document are being displayed. The primary example of this are the index, or listing, pages.
+
+For example, if a `team` piece type has a relationship field that connects it to many `players` (as in baseball "players"), you may want to display all of the players' information on the individual team page (the [show page](/reference/glossary.md#show-page)). However, on the [index page](/reference/glossary.md#index-page) for all teams, you are not displaying all players for every team, so it lightens the server load to ignore the relationship field in that context. So you might configure the field as such:
+
+```javascript
+// `_players` will only be populated when only one team document is being fetched
+_players: {
+  label: 'Players',
+  type: 'relationship',
+  withType: 'player',
+  ifOnlyOne: true
+}
+```
+
+## Populated nested relationships using `withRelationship`
```

Comment:
Should be plural

Reply:
What should be plural? I'm switching "populated" to "populating".